src/components/BlogPage/BlogPage.spec.js

```diff
@@ -1,32 +1,55 @@
-import React from 'react';
-import MockAdapter from 'axios-mock-adapter';
-import axios from 'axios';
-import { shallow } from 'enzyme';
+import React from "react";
+import MockAdapter from "axios-mock-adapter";
+import axios from "axios";
+import { shallow } from "enzyme";
 
-import BlogPage from './BlogPage';
-import mockedData from '../../mockedDataForTests';
+import BlogPage from "./BlogPage";
+import mockedData from "../../mockedDataForTests";
 
-describe('<BlogPage />', () => {
+describe("<BlogPage />", () => {
```

Comment:
What is the rule for quotes in your **IDE** ?

Reply:
it is doing the prettier extension, it should be a single quote?

Reply:
Looks like that:
`"prettier.singleQuote": true`